cime_config/machines/config_machines.xml

```diff
@@ -1283,6 +1282,7 @@
       <env name="OMP_STACKSIZE">64M</env>
       <env name="OMP_PROC_BIND">spread</env>
       <env name="OMP_PLACES">threads</env>
+      <env name="PATH">/ascldap/users/jgfouca/packages/openmpi-4.1.4/bin:$ENV{PATH}</env>
```

Comment:
Can we put the src/build/install folders somewhere in something like `/sems-data-store/ACME/software/openmpi`?
That way, if you happen to be on a long vacation when a critical bug shows up, we can still update the installation without the need of a PR in e3sm.

I'm including the build dir, since I'm hoping you left a `do-configure.sh` kind of script there, which we can then use if we need to adjust the installation...